sdk/cognitiveservices/cognitiveservices-personalizer/src/personalizerClientContext.ts

```diff
@@ -42,7 +41,7 @@ export class PersonalizerClientContext extends msRest.ServiceClient {
 
     super(credentials, options);
 
-    this.baseUri = "{Endpoint}/personalizer/v1.0";
+    this.baseUri = "{Endpoint}/personalizer/v1.1-preview.1";
```

Comment:
Since we are moving to a preview version of the API, please check with the service team and other languages if we are planning to have the SDK release as GA or preview

It would be good to have a manual entry in the readme file telling the difference so that people are aware. See https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/cognitiveservices/cognitiveservices-luis-authoring/README.md for example. I know this will have to be manually fixed up again the next time we generate the package, but it is worth the work if we can give some clarity to the users

Reply:
Done